WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -60,4 +60,9 @@ final class OrdersTests: XCTestCase {
             .cancelOrderCreation()
             .verifyOrdersScreenLoaded()
     }
+
+    func test_load_orders_universal_link() throws {
+        try ExternalAppScreen().openUniversalLinkFromSafariApp(linkedScreen: "orders")
+        try SingleOrderScreen().verifySingleOrderScreenLoaded()
```

Comment:
Since the link that is tapped contains the order ID, I think it's critical to check that the correct order ID is opened. In fact, on iPad, I noticed multiple instances of other order being opened (probably due to slow load of the list):

https://user-images.githubusercontent.com/73365754/231762533-ba2a87f4-fd3a-4aca-9682-bef6429c35a5.mov

The test still passes. The fact of wrong order being opened is likely an app bug, it's not a test bug. But the test should catch this.

Reply:
good catch! looks like it's a bug on iPad, but i did try on an iPad simulator manually and wasn't able to reproduce it. it could be a mock-related bug that i have not figured out. so i'm skipping the test for iPad for now (after the assertions were updated, test on iPad was failing)

the code is updated to include better assertions in https://github.com/woocommerce/woocommerce-ios/pull/9442/commits/cee362d38e03dde9ee7d567d2a056d40c7f0e38d i'll add in separate comments why some of the changes were made but the tldr is because the order number used for this test and `test_load_orders_screen` are different